### PR TITLE
add NacosEnv document

### PR DESCRIPTION
**Pull request by onewe**
## What is the purpose of the change

XXXXX

## Brief changelog

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/nacos-group/nacos-group.github.io/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Test your code locally by running `docsite start`, and make sure it works as expected.
- [ ] Make sure no files under build directory is added.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: docs/zh-cn/sdk.md [hunk 1]
```diff
@@ -17,6 +17,69 @@ Maven 坐标
 </dependency>
 ```
 
+## Nacos Environment(目前还未投入使用)
+### 描述
+为了统一 `Nacos sdk` 获取参数的方式,社区提供了`NacosEnvs`工具类(目前还未正式投入使用), 详见: https://github.com/alibaba/nacos/issues/8622 .该工具支持从系统环境变量、命令行参数、用户自定义参数和默认参数中获取值.
```
Comment (KomachiSion) [inline docs/zh-cn/sdk.md:22]: (目前还未正式投入使用)

改成提供了beta测试功能

File: docs/zh-cn/sdk.md [hunk 2]
```diff
@@ -17,6 +17,69 @@ Maven 坐标
...
+<img src="/img/nacos_env_default_order.png" alt="nacos_env_default_order" width = "400" height = "500"></br>
+</p>
+
+上图是默认查找顺序: `PROPERTIES->SYS->JVM->DEFAULT_SETTING`,若没查找到指定参数的值,则依顺序向下查找,直至查找到为止.
+
+### 设计
+![nacos_env_design](/img/nacos_env_design.png)
+### 调整查找优先级
+为了灵活切换优先级, 提供了`nacos.envs.search`参数,可用于系统环境变量和命令行参数中. 可接受值为:
+- DEFAULT_SETTING 优先从默认参数配置中查找 `(DEFAULT_SETTING->PROPERTIES->SYS->JVM)`
```
Comment (KomachiSion) [inline docs/zh-cn/sdk.md:33]: 我们有这种规则吗？ 这个规则不太合理

File: docs/zh-cn/sdk.md [hunk 3]
```diff
@@ -17,6 +17,69 @@ Maven 坐标
...
+| nacos.server.port | 8848 |
+| NACOS.CONNECT.TIMEOUT | 1000 |
+| PER_TASK_CONFIG_SIZE | 3000 |
+
+### NacosEnvs 的使用
+```java
+// 使用前需进行初始化
+final Properties properties = new Properties();
+properties.setProperty("nacos.home", "/home/nacos");
+NacosEnvs.init(properties);
```
Comment (KomachiSion) [inline docs/zh-cn/sdk.md:78]: 用户需要自己进行初始化吗？ 我理解是在创建NacosConfigServer或者NacosNamingServer的时候，nacosclient自动初始化的吧。

Comment (onewe) [reply docs/zh-cn/sdk.md:78]: 这个的确不需要用户初始化, 但是 NacosConfigServer 和 NacosNamingServer 这块儿并没有改成 从 `NacosEnvs ` 获取.

File: docs/zh-cn/sdk.md (hunk 2 above)
Comment (onewe) [reply docs/zh-cn/sdk.md:33]: default 这个的确没有 我写错了 我改改

File: docs/zh-cn/sdk.md (hunk 1 above)
Comment (onewe) [reply docs/zh-cn/sdk.md:22]: ok

**Pull request closed without merging**